cookiecutter.json

```diff
@@ -1,8 +1,9 @@
 {
     "full_name": "Name or Organization",
```

Comment:
since this is asking for org, why don't we call the key "org_name"?  or will that break things too much?

Reply:
I didn't want to touch these names, but I can open a new issue addressing the names in the `.json`.